cloud/pkg/cloudstream/containerexec_connection.go

```diff
@@ -21,6 +21,7 @@ type ContainerExecConnection struct {
 	Conn         net.Conn
 	session      *Session
 	edgePeerStop chan struct{}
+	closeChan    chan bool
```

Comment:
We can use sync.Once to make sure the edgePeerStop channel is closed only once

Reply:
According to the channel closing principle, one general principle of using Go channels is don't close a channel from the receiver side and don't close a channel if the channel has multiple concurrent senders.
It will be more graceful to close a channel from the receiver side.